client-js/app/elements/elements.html

```diff
@@ -25,6 +26,8 @@
 <link rel="import" href="../bower_components/paper-material/paper-material.html">
 <link rel="import" href="../bower_components/paper-menu/paper-menu.html">
 <link rel="import" href="../bower_components/paper-styles/paper-styles-classes.html">
+<link rel="import" href="../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../bower_components/paper-radio-button/paper-radio-button.html">
```

Comment:
I'd prefer to import these where they are used, instead of in this global file. That is, put these imports in `labrad-plot.html` directly. It's unfortunate that we inherited this bad habit from the polymer example project...